pkg/service/endpoints_controller.go

```diff
@@ -56,10 +56,10 @@ func (e *EndpointController) SyncServiceEndpoints() error {
 			continue
 		}
 
-		glog.V(3).Infof("About to update endpoints for service %v", service.Name)
+		glog.V(5).Infof("About to update endpoints for service %s/%s", service.Namespace, service.Name)
```

Comment:
We should land on one syntax for printing namespace and name (for all sorts of objects) "namespace/name" vs "name.namespace" vs other.

namepsace/name has an advantage when both name and namespace are DNS subdomains: "foo.bar/quux.zorb" is clear, while "quux.zorb.foo.bar" is ambiguous.

If we all agree, we should start enforcing it in code reviews and fixing it up when we find it.